assign.py

```diff
@@ -206,9 +206,9 @@ def main():
         elif options.sites == "t2":
             sites = T2_SITES
         else:
-            sites = SI.sites_ready
+            sites = [ s for s in SI.sites_ready if  (s.startswith('T1_') or s.startswith('T2_'))]
     else: 
-         sites = SI.sites_ready
+        sites = [ s for s in SI.sites_ready if  (s.startswith('T1_') or s.startswith('T2_')]
```

Comment:
I am trying to use the script, and the first error I got is a "SyntaxError: invalid syntax" in this line.